webpack.config.js

```diff
@@ -60,7 +60,7 @@ const paths = {
  */
 const prod = {
   mode: 'production',
-  devtool: 'inline-source-map',
+  devtool: 'source-map',
```

Comment:
+1. this may cause some issues with debugging, especially because of our currently convoluted build process (aegir/tsc->webpack) but will definitely reduce prod bundle size.

Reply:
Why do we use both aegir and webpack for this repo. Aegir comes with esbuild, shouldn't esbuild serve us?